README.md

```diff
@@ -21,6 +21,20 @@ from `https://api.github.com/repos/jaemk/self_update/releases/latest`.
 Note, the [`trust`](https://github.com/japaric/trust) project provides a nice setup for
 producing release-builds via CI (travis/appveyor).
 
+### Features
```

Comment:
can you move this to the lib.rs file and run the `readme.sh` script (requires `cargo-readme`)?

Reply:
Fixed in e46d083ddf74ac695f3885bdcea09edb.